### DISCUSSION TITLE
Suggestion: Create a Matrix room for general Chatwoot discussion

### DISCUSSION THREAD

**Discussion by pucilpet** [chatwoot/chatwoot#8493] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
You could create a room/channel on a public network (matrix.org for example - _easy_) or you can self-host a matrix server (_more control_ but some effort to get started)

**Describe alternatives you've considered**
Your Discord - but it's not FOSS and many people dislike using it.

**Additional context**
https://matrix.org/docs/guides/getting-involved

If you go for the self-hosted server, this ansible playbook makes it really easy to get started:
https://github.com/spantaleev/matrix-docker-ansible-deploy/

**Comment by pranavrajs** [comment]
@pucilpet This is something we want to explore on, there is a lot of interest going on in Matrix. This wouldn't be prioritized immediately, but it is something I personally like to explore on. If you have some pointers on where to start, it would be nice.

**Comment by pucilpet** [comment]
> @pucilpet This is something we want to explore on, there is a lot of interest going on in Matrix. This wouldn't be prioritized immediately, but it is something I personally like to explore on. If you have some pointers on where to start, it would be nice.

Well it all depends on which route you decide to go for - a self hosted server or a matrix.org room.

If you decide to try the self hosted one and are some what familiar with ansible, that playbook I linked is a super easy way to get started and you can get a basic setup (just synapse server and matrix registration for Example) up and running in less than 15 minutes. When you federate the server, people can join from other homeservers (e.g. Matrix.org) with their accounts and you can only allow registration to your homeserver for people in your organization (with a matrix registration tokens for Example).

 If and when you are ready to explore it and if you need any help... feel free to ping me.
